sql/core/src/main/scala/org/apache/spark/sql/streaming/progress.scala

```diff
@@ -200,7 +202,7 @@ class SourceProgress protected[sql](
  */
 @InterfaceStability.Evolving
 class SinkProgress protected[sql](
-    val description: String) extends Serializable {
```

Comment:
Hi, @jaceklaskowski . For this, the original one looks correct for me.

Reply:
Really?! Then the other places in the file are incorrect like https://github.com/jaceklaskowski/spark/blob/337ad489bb43ef93a651bdd4952bd7f0738698dc/sql/core/src/main/scala/org/apache/spark/sql/streaming/progress.scala#L90 or https://github.com/jaceklaskowski/spark/blob/337ad489bb43ef93a651bdd4952bd7f0738698dc/sql/core/src/main/scala/org/apache/spark/sql/streaming/progress.scala#L161? I might be missing something though.

Reply:
Yes. Please refer here, https://github.com/databricks/scala-style-guide
```scala
class Foo(
    val param1: String,  // 4 space indent for parameters
    val param2: String,
    val param3: Array[Byte])
  extends FooInterface  // 2 space indent here
  with Logging {

  def firstMethod(): Unit = { ... }  // blank line above
}
```

Reply:
Should I fix the other places then?

Reply:
Sorry, I'm not in a position to say that. Maybe, committers will review this and give a direction.

Reply:
not a committer but would like to leave this suggestion : 
- codestyle changes are orthogonal to the motive of the PR and should be done separately. Generally, every PR should address one problem and not have changes unrelated to it. In event of revert or bisecting commits to pin-point regression, following this practice helps a lot.
- It would be beneficial to see why checkstyle does not catch such instances and fix that (along with making all such instances consistent with the rules). Otherwise this would be a one off fix and we would continue to pile up similar inconsistencies in future development without anyone realising this.

Reply:
I am a committer. Generally, a PR should targets an issue without other changes. I use common sense - fixing styles around the changes is fine if it does not look making revert/backport harder.